examples/b2c/checkout/integrations/classes/B2BTaxSample.cls

```diff
@@ -34,14 +35,25 @@ global class B2BTaxSample implements sfdc_checkout.CartTaxCalculations {
                     // For simplicity and testing purposes, we just assign some SKU to this charge so that the taxation external service returns some value.
                     cartItemSKU = 'ChargeSKU';
                 }
-                cartItemIdsBySKU.put(cartItemSKU, cartItem.Id);
-                cartItemTotalPriceBySKU.put(cartItemSKU, cartItem.TotalPrice);
+
+                String cartItemId = cartItem.Id;
+                cartItemIdsBySKU.put(cartItemSKU, cartItemId);
+
+                List<CIPA> cipaList = new List<CIPA>();
+                
+                for(CartItemPriceAdjustment cipa : [SELECT Id, TotalAmount FROM CartItemPriceAdjustment WHERE CartItemId = :cartItemId]) {
```

Comment:
how about target type?

Reply:
We don't need to pass Target Type to the external service.